grid_map_cmake_helpers/package.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>grid_map_cmake_helpers</name>
+  <version>2.0.0</version>
+  <description>CMake support functionality used throughout grid_map</description>
+  <maintainer email="someone@gmail.com">who_is_the_maintainer</maintainer>
```

Comment:
Can you add me mwulf@anybotics.com "Maximilian Wulf", ynava@anybotics.com "Yoshua Nava" and Steve as maintainers.